pages/common/colordiff.md

```diff
@@ -0,0 +1,25 @@
+# colordiff
+
+> A tool to colorize diff diff output.
+> The Perl script colordiff is a wrapper for `diff` and produces the same output but with pretty 'syntax' highlighting. Colour schemes can be customized..
+> More information: <https://github.com/kimmel/colordiff>.
+
+- Compare files:
+
+`colordiff {{file-1}} {{file-2}}`
+
+- Compare files, output in two columns:
+
+`colordiff -y {{file-1}} {{file-2}}`
+
+- Compare files, Ignore case differences in file contents:
```

Comment:
How about- "Compare files ignoring case differences in file contents"

Reply:
it make sense and I have removed "compare files" content for all examples except first.